dev/maven-plugin/src/main/maven/plugin.xml

```diff
@@ -98,6 +98,13 @@
             <editable>true</editable>
             <description>The JVM options to pass to Cassandra.</description>
         </parameter>
+        <parameter>
+            <name>cassandraYAMLFile</name>
```

Comment:
Maybe `cassandraYamlFile` for consistency with `cassandraJvmOptions`?

Reply:
yep, I will change it.